cmd/namespace.go

```diff
@@ -86,13 +86,11 @@ Please note that at this time all namespaces created in the registry are world-r
 		},
 		Args:        cobra.RangeArgs(1, 3),
 		Annotations: make(map[string]string),
-		Example: `  circleci namespace create NamespaceName github OrgName
+		Example: `  circleci namespace create NamespaceName --org-id "your-org-id-here"
```

Comment:
I suppose this example was showing the two ways you could use the command which were `circleci namespace create NamespaceName <vcs-type> <org-name>` or `circleci namespace create NamespaceName --orgId <org-id>`.

This PR removes the first way of using the command so you should simply remove the first example instead of changing it.